src/cli/commands/snip.test.ts

```diff
@@ -325,4 +325,51 @@ struct ContentView: SwiftUI.App {
  `);
  */
   });
+  it("handles the --id option", async () => {
+    const text = `
+public class Main {
+  public static void main(String[] args){
+    // :snippet-start: test-block
+    System.out.println("1");
+    // :snippet-end:
+    // :snippet-start: test-block-2
+    System.out.println("2");
+    // :snippet-end:
+  }
+}
+`;
+    const rootPath = "/path/to/project";
+    const snippetName = "test.codeblock.test-block.java";
+    const destinationPathLocal = "/stateAndEmphasize/local";
+    const testFileName = "test.java";
+
+    await System.fs.mkdir(rootPath, {
+      recursive: true,
+    });
+    await System.fs.mkdir(destinationPathLocal, {
+      recursive: true,
+    });
+    await System.fs.writeFile(Path.join(rootPath, testFileName), text, "utf8");
+
+    await snip({
+      paths: [rootPath],
+      destination: destinationPathLocal,
+      id: "test-block",
+    });
+
+    let fileContentsSync = await System.fs.readFile(
```

Comment:
Thanks for the note. I agree this should be const instead of let. 

Going forward, I will try to pay attention to use const/let more thoroughly in JS programming.